integrations/snyk/snyk/client.py

```diff
@@ -105,15 +106,24 @@ async def get_issues(self, org_id: str, project_id: str) -> list[dict[str, Any]]
             return cache
 
         url = f"{self.api_url}/org/{org_id}/project/{project_id}/aggregated-issues"
-        issues = (
-            await self._send_api_request(
-                url=url,
-                method="POST",
-                version=self.snyk_api_version,
-            )
-        )["issues"]
-        event.attributes[cache_key] = issues
-        return issues
+        try:
+            issues = (
+                await self._send_api_request(
+                    url=url,
+                    method="POST",
+                    version=self.snyk_api_version,
+                )
+            )["issues"]
+            event.attributes[cache_key] = issues
+            return issues
+        except httpx.HTTPStatusError as e:
+            if json.loads(e.response.text).code == "SNYK-9999":
+                logger.error(
+                    f"Encountered Synk internal error while getting issues "
+                    f"for project {project_id} in org {org_id}."
+                )
+                return []
+            raise
```

Comment:
this error can occur on every request, maybe we should do it inside the `self._send_api_request`